unreleased/clickhouseexporter-doc.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: clickhouseexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "update table schema"
+
+# One or more tracking issues related to the change
+issues: [8028]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  1. add performance guide.
+  2. add more SQL example.
```

Comment:
The log entry is not needed since it's just a documentation update not user facing change

Reply:
I mean let's remove this file completely. We don't need a changelog entry for doc updates

Reply:
i see, fixed.